src/ECCX08.cpp

```diff
@@ -148,7 +148,7 @@ int ECCX08Class::random(byte data[], size_t length)
       return 0;
     }
 
-    int copyLength = min(32, length);
+    int copyLength = min(32, (int)length);
```

Comment:
Why is this cast needed?

Reply:
To suppress a warning about unsigned vs signed comparison.